src/query/sql/src/planner/binder/binder.rs

```diff
@@ -663,6 +664,22 @@ impl<'a> Binder {
                 self.bind_set_priority(priority, object_id).await?
             },
         };
+
+        match plan.kind() {
+            QueryKind::Query { .. } | QueryKind::Explain { .. } => {}
+            _ => {
+                let meta_data_guard = self.metadata.read();
+                let tables = meta_data_guard.tables();
+                for t in tables {
```

Comment:
If a database has many tables, this check may break performance?

Reply:
This check is work?
https://github.com/datafuselabs/databend/blob/2bfe458d31feb6a067cefb3b090c02f260d0fbcb/src/query/sql/src/planner/binder/table.rs#L265-L269

Reply:
this only involves the tables that bounded in the statement, not all the tables of a database.